sdk/core/core-tracing/test/cache.spec.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import assert from "assert";
+import { GLOBAL_TRACER_VERSION } from "../src/utils/cache";
+import * as fs from "fs";
+import * as path from "path";
+
+describe("cache", () => {
+  const packageJson = JSON.parse(
+    fs.readFileSync(path.join(__dirname, "../package.json"), { encoding: "utf8" })
+  );
+
+  it("ensure version matches @opentelemetry/api version", () => {
```

Comment:
per above comment, we can delete this whole constant and then we don't need a test for it